<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>316</width>
    <height>357</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Relatório Extract v1.0</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="Bdados">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>62</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Carregar Dados/Planilha</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Bimagens">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>90</y>
      <width>151</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Carregar Imagens</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>281</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Emitir relatórios de todo os dados ?</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="Ssim">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>170</y>
      <width>61</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>SIM</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="Snao">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>170</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>NÃO</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Tlinha">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>190</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="toolTip">
     <string comment="555" extracomment="- para selcionar o range e ; para valores individuais EX.: 1 - 5 (1 a 5) | 1;3;5 (linhas 1 3 5 serão emitidas)">- para selcionar o range e ; para valores individuais EX.: 1 - 5 (1 a 5) | 1;3;5 (linhas 1 3 5 serão emitidas)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>210</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Selecionar linhas</string>
    </property>
   </widget>
   <widget class="QLabel" name="Status_dados">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>70</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="Status_imagens">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>100</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>271</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cascadia Code</family>
      <pointsize>16</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Relatório Extract v1.0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Bemissao">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>250</y>
      <width>171</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Emitir Relatórios</string>
    </property>
   </widget>
   <widget class="QLabel" name="Status_emissao">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>280</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="Bardados">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>66</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
    <property name="textVisible">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QProgressBar" name="Barimagens">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>100</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
    <property name="textVisible">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>316</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="menuConfig"/>
   </widget>
   <addaction name="menuMenu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="menuConfig">
   <property name="text">
    <string>Configurações</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
